dlt/common/normalizers/json/relational.py

```diff
@@ -505,6 +504,10 @@ def _get_row_id_type(
                 if x_row_version_col == DataItemNormalizer.C_DLT_ID:
                     return "row_hash"
         elif _r_lvl > 0:  # child table
+            merge_strategy = DataItemNormalizer._get_merge_strategy(schema, table_name)
+            if merge_strategy in ("upsert", "scd2"):
```

Comment:
do I understand correctly, that one has to set merge strategy to "upsert" explicitely when using merge with delta tables on the filesystem destination? If that is the case, it would be nice to have a mechanism that does it automatically.